app/frontend/components/domains/single-zone-cooling-heating-tool/form-section/review-form.tsx

```diff
@@ -0,0 +1,180 @@
+import { Box, Button, Divider, Flex, Grid, GridItem, Heading, Link, Text } from "@chakra-ui/react"
+import { observer } from "mobx-react-lite"
+import React from "react"
+import { useFormContext } from "react-hook-form"
+import { useTranslation } from "react-i18next"
+import { useMst } from "../../../../setup/root"
+
+type KeyPath = string
+
+const useLabels = () => {
```

Comment:
Do you see a way to obviate this entire section? Do not clutter the codebase with overly verbose code when a small piece suffices.